ppr-ui/src/components/mhrTransfers/DeathCertificate.vue

```diff
@@ -139,9 +138,10 @@ export default defineComponent({
     }
 
     // Validate form when prompted
-    watch(() => props.validate, (validate: boolean) => {
+    watch(() => props.validate, async (validate: boolean) => {
+      await nextTick()
       validate && (context.refs.deathCertificateForm as FormIF).validate()
-    })
+    }, { immediate: true })
```

Comment:
Once review confirm has been selected, we want to prompt field validation each time we open Death Cert Form .